CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Contributing to AdGuard filters
+
+If you want to make AdGuard better by implementing new rules, follow the instructions below to make your ideas come to life faster!
+
+## How to work with the repo
+
+1. Install [Visual Studio Code](https://code.visualstudio.com/download) with the ["Adblock" syntax plugin](https://marketplace.visualstudio.com/items?itemName=adguard.adblock).
+2. [Clone](https://docs.github.com/repositories/creating-and-managing-repositories/cloning-a-repository#cloning-a-repository) (or fork and clone) this repo.
+3. Make your changes and commit them.
+4. Each commit should be linked to an issue. The commit message should look like the examples below. That way, the issue will be closed automatically once the commit is merged into the `master` branch.
+   * `Fix #123 site` where `123` is the issue number and `site` - website from the issue.
+   * `Fix #123 comment` where `comment` is your comment for additional fixes in the pull request.
+
+## How to write filter rules
+
+Before you begin, please read and understand the current [filters policy](https://kb.adguard.com/general/adguard-filter-policy) we adhere to. One of its most important points is the [quality requirements](https://kb.adguard.com/general/adguard-filter-policy#quality-requirements-for-filtering-rules).
```

Comment:
@zloyden should we update the quality requirements and maybe list them here?

As I understand, we should at least have different quality requirements for the rules that hide ad placeholders (there's no need to apply them on websites that are not very popular).

Reply:
> should we update the quality requirements and maybe list them here?

Maybe not, those statements are clear to me. @Alex-302 wanted to add some changes in general.

> there's no need to apply them on websites that are not very popular

Probably, I missed something. Can you please expain? We don't add any type of the rules if the website has low visitors, except cases with incorrect blocking or adblock detection.

Reply:
There're some reports on ad placeholders left after blocking ads. These should be only accepted if the website is popular enough, it makes little sense to handle placeholders on websites with 30k monthly visitors.

Reply:
> it makes little sense to handle placeholders on websites with 30k monthly visitors
Ok, I got it. This one should be added in the filters policy. However, what should be the acceptable value of visitors in this case?

Reply:
That's up to you all to decide